src/Sylius/Bundle/UserBundle/Form/Type/UserType.php

```diff
@@ -38,8 +46,31 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
             ])
             ->add('enabled', CheckboxType::class, [
                 'label' => 'sylius.form.user.enabled',
+                'required' => false,
+            ])
+            ->add('verifiedAt', CheckboxType::class, [
+                'label' => 'sylius.form.user.verified',
+                'required' => false,
             ])
         ;
+
+        $builder->get('verifiedAt')->addModelTransformer(new UserVerificationTransformer(), true);
+
+        $builder->addEventListener(FormEvents::POST_SET_DATA, static function(FormEvent $event) {
+            /** @var ShopUser|null $data */
+            $data = $event->getData();
+            if (!$data) {
```

Comment:
Can you use a more strict comparison here?
```suggestion
            if (null === $data) {
```